homeassistant/components/notion/__init__.py

```diff
@@ -246,6 +248,9 @@ def _async_update_from_latest_data(self) -> None:
     @callback
     def _handle_coordinator_update(self):
         """Respond to a DataUpdateCoordinator update."""
+        if self.task_id not in self.coordinator.data["tasks"]:
+            return
+
         self.hass.async_create_task(self._async_update_bridge_id())
         self._async_update_from_latest_data()
```

Comment:
You still need to write the state, as it's now unavailable. 
```suggestion
        if self.task_id not in self.coordinator.data["tasks"]:
            self.hass.async_create_task(self._async_update_bridge_id())
            self._async_update_from_latest_data()

```

Reply:
I think you mean:

```
if self.task_id in self.coordinator.data["tasks"]:
    self.hass.async_create_task(self._async_update_bridge_id())
    self._async_update_from_latest_data()
```

👍